satellite/satellitedb/irreparabledb.go

```diff
@@ -41,6 +41,7 @@ func (db *irreparableDB) IncrementRepairAttempts(ctx context.Context, segmentInf
 		dbxInfo.RepairAttemptCount++
 		updateFields := dbx.Irreparabledb_Update_Fields{}
 		updateFields.RepairAttemptCount = dbx.Irreparabledb_RepairAttemptCount(dbxInfo.RepairAttemptCount)
+		updateFields.SegDamagedUnixSec = dbx.Irreparabledb_SegDamagedUnixSec(segmentInfo.RepairUnixSec)
```

Comment:
How does this change relate to the test case?

Reply:
I asked @aligeti and he said that repair attempt time should always be updated with attempt count. Turns out that it wasn't updated.